x-pack/plugins/ml/public/application/supplied_configurations/supplied_configurations_flyout/overview_tab_content.tsx

```diff
@@ -0,0 +1,342 @@
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiNotificationBadge,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  EuiToolTip,
+} from '@elastic/eui';
+import {
+  usePermissionCheck,
+  createPermissionFailureMessage,
+} from '../../capabilities/check_capabilities';
+import type { Module, RecognizeModuleResult } from '../../../../common/types/modules';
+import { useEnabledFeatures } from '../../contexts/ml/serverless_context';
+import { useMlKibana } from '../../contexts/kibana';
+import type { TabIdType, KibanaAssetType } from './flyout';
+import { TAB_IDS } from './flyout';
+import { DataViewsTable } from './data_views_table';
+import { isLogoObject } from '../supplied_configurations';
+
+export const LABELS = {
+  dashboard: (
+    <FormattedMessage
+      id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.dashboardLabel"
+      defaultMessage="Dashboard"
+    />
+  ),
+  jobs: (
+    <FormattedMessage
+      id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.jobsLabel"
+      defaultMessage="Jobs"
+    />
+  ),
+  search: (
+    <FormattedMessage
+      id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.searchLabel"
+      defaultMessage="Search"
+    />
+  ),
+  visualization: (
+    <FormattedMessage
+      id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.visualizationLabel"
+      defaultMessage="Visualization"
+    />
+  ),
+};
+
+export type LabelId = keyof typeof LABELS;
+
+const ListDescriptionItem = ({
+  label,
+  onClick,
+  value,
+}: {
+  label?: LabelId;
+  value: number | string;
+  onClick?: () => void;
+}) => (
+  <>
+    <EuiDescriptionListDescription>
+      {label ? (
+        <EuiButtonEmpty color={'text'} onClick={onClick}>
+          {LABELS[label]}
+        </EuiButtonEmpty>
+      ) : (
+        ''
+      )}
+      {label ? ' ' : ''}
+      <EuiNotificationBadge size={label ? 's' : 'm'} color="subdued">
+        {value}
+      </EuiNotificationBadge>
+    </EuiDescriptionListDescription>
+    {label ? null : <EuiSpacer size="s" />}
+  </>
+);
+
+interface Props {
+  module: Module;
+  setSelectedTabId: (tabId: TabIdType) => void;
+  setSelectedKibanaSubTab: (kibanaSubTab: KibanaAssetType) => void;
+}
+
+export const OverviewTabContent: FC<Props> = ({
+  module,
+  setSelectedTabId,
+  setSelectedKibanaSubTab,
+}) => {
+  const [runningDataRecognizer, setRunningDataRecognizer] = useState<boolean>(false);
+  const [recognizerWasRun, setRecognizerWasRun] = useState<boolean>(false);
+  const [matchingDataViews, setMatchingDataViews] = useState<RecognizeModuleResult>([]);
+  const {
+    services: {
+      docLinks,
+      mlServices: {
+        mlApi: { recognizeModule },
+      },
+    },
+  } = useMlKibana();
+  const logsConfigsUrl = docLinks.links.ml.logsAnomalyDetectionConfigs;
+  const metricsConfigsUrl = docLinks.links.ml.metricsAnomalyDetectionConfigs;
+  const canCreateJob = usePermissionCheck('canCreateJob');
+  const { showContextualInsights: isServerless } = useEnabledFeatures();
```

Comment:
Can we add a new feature to the `EnabledFeatures` object for this. We were asked by core to try not to have any `isServerless` variables in the code and instead have specific feature flags. Although some `isServerless` variables have snuck in, it would be good to not add any more.

Reply:
Ah, good point - updated in c95eeaac6e88ebed1292d54ceeea5a6babbc65d1